### ISSUE TITLE
Dump fastq to sra-lite 

### ISSUE THREAD

**Issue by ledwmp** [ncbi/sra-tools#995]
Is there documentation on which flags to use to dump fastq to sra-lite format?

We are trying to standardize some of our pipelines, and have some data in sra-lite format and others as fastq. It seems using latf-load followed by kar is probably the right path, but it's unclear how to run a conversion from fastq to sra-lite.

Thanks, any help would be appreciated.

**Comment by durbrow** [comment]
You are correct. The generation of the sra-lite files happens inside the SRA ingest pipeline. It uses the toolkit but it isn't a part of the toolkit and the toolkit team is not qualified to describe the intricacies of the process (IOW it is complicated and I don't know all of the steps involved.)

We may be able to modify `latf-load` to produce the sra-lite output directly. If you were to request it and aren't interested in the full quality version. (It is mostly the removal of the quality column after ensuring that we record that the read was low quality.)

Please let us know. We are interested in helping you with this.

**Comment by ledwmp** [comment]
Adding that functionality to latf-load would be great. 

It's a little unclear in the documentation when the vendor flag is used to set the pass|filter flag, when quality scores are used to set the flag, and how these filters work. Otherwise, we would probably attempt to implement these filters ourselves.

Adding this functionality to latf-load would ensure consistency with the sra-lite format. We appreciate your help!

**Comment by durbrow** [comment]
Sorry I didn't get back to you sooner. This might help with what rules are used for updating the vendor flag. See tools/internal/make-read-filter/make-read-filter.c Near the top of the file is a comment with the rules.
